src/ui/components/Scanner/Scanner.tsx

```diff
@@ -115,6 +116,15 @@ const Scanner = forwardRef(
       dispatch(setMultiSigGroupCache(newMultiSigGroup));
     };
 
+    const handleMultisigConnections = async (groupId: string) => {
+      const connections =
+        await Agent.agent.connections.getMultisigLinkedContacts(groupId);
+
+      connections.forEach((connection) => {
+        dispatch(updateOrAddMultisigConnectionCache(connection));
+      });
+    };
```

Comment:
Can `connectByOobiUrl` return enough information to just construct the `ConnectionShortDetails` for the new connection and add it? Rather than fetching _all_ of these contacts from the DB and re-adding